### ISSUE TITLE
[Bug] Unexpected possible-variable-overwriting when elevating the scope of an already existing variable

### ISSUE THREAD

**Issue by danielherrera** [MarketSquare/robotframework-robocop#1053]
### What happened?

I was trying to clean some **replace-set-variable-with-var** I have in my test project. Initially it was:

```
*** Test Cases ***`
${date}=    Get Invoice Date
Set Suite Variable    ${DATE}    ${date}
```

So I moved to the new VAR syntax to get rid of the problem like this:

```
*** Test Cases ***
${date}=    Get Invoice Date
VAR    ${DATE}    ${date}    scope=SUITE
```

But now I am getting **possible-variable-overwriting** so I was wondering what should I use to elevate the scope of an existing variable? Not sure if it is a bug but any suggestion is appreciated.

### What command/code did you try to run?

Regular command execution.

### What is the full error message?

There is no error message, the regular finding messages will be printed.

### What did you expect to happen instead?

All warnings to be corrected.

### Operating System

Windows

### Robocop version

5.0.3

**Comment by bhirsz** [comment]
Oh. I didnt consider simply rescoping the variable when updating the rules for Robot Framework 7. In your case it should warn anything. I think possible-variable-overwritting should only inspect VAR with local scope to avoid it
